### DISCUSSION TITLE
How can I create bot that replies in less than 1 minute?

### DISCUSSION THREAD

**Discussion by legendaryangelist** [tweepy/tweepy#2127] [Questions] [answered]
Hello.
I have Basic access level.
I'm going to create bot replies in less than 1 minute whenever users mention the bot and ask.
So I have to retrieve mentions for the bot every 1 minute.
But I found rate limit for get_mentions_user endpoint is 5 times per 15 min.
So is there any solution to avoid rate limit?
Please help me.

**Comment by Harmon758** [comment, marked as answer]
I'm not sure what "get_mentions_user endpoint" you're referring to, but the rate limit for [the GET /2/users/:id/mentions endpoint](https://developer.twitter.com/en/docs/twitter-api/tweets/timelines/api-reference/get-users-id-mentions) that [`Client.get_users_mentions`](https://tweepy.readthedocs.io/en/v4.14.0/client.html#tweepy.Client.get_users_mentions) uses should be 10 requests / 15 mins per user and 15 requests / 15 mins per app with Basic access, according to the Developer Portal.

Alternatively, you can search for the mention as well.
